import_claims/crowdsupply/import.py

```diff
@@ -0,0 +1,146 @@
+import validators
+import urllib
+import pandas as pd
+import sys
+import psycopg2
+import json
+from dotenv import dotenv_values
+from datetime import datetime, timedelta
+
+config = dotenv_values("../.env")
+
+
+def print_error_and_exit(str):
+    print(f"Error: {str}")
+    sys.exit(2)
+
+
+def get_db_connection():
+    conn = psycopg2.connect(config.get("DATABASE_URL"))
+    return conn
+
+
+def db_get_one(query: str):
+    conn = None
+    res = None
+    try:
+        conn = get_db_connection()
+        cur = conn.cursor()
+
+        cur.execute(query)
+        res = cur.fetchone()
+
+        cur.close()
+    except (Exception, psycopg2.DatabaseError) as error:
+        print(error)
+    finally:
+        if conn is not None:
+            conn.close()
+    return res
+
+
+def db_post_many(query, values):
+    try:
+        conn = get_db_connection()
+        cursor = conn.cursor()
+
+        cursor.executemany(query, values)
+        conn.commit()
+
+    except (Exception, psycopg2.Error) as error:
+        print("Failed insertion: {}".format(error))
+
+    finally:
+        if conn:
+            cursor.close()
+            conn.close()
+
+
+def main():
+    settings = None
+
+    try:
+        with open("config.json") as f:
+            settings = json.load(f)
+    except Exception as e:
+        print_error_and_exit(str(e))
+
```

Comment:
you shouldn't need to copy this

Reply:
the whole idea of the config is that you can use common code?  the idea is to run `addclaims.py` with a parameter

you shouldn't copypasta same code, if you need some different can refactor but the idea is with the config file, to just have the one import script not a bunch of copies

Reply:
Thank you. But I had some logics in some of the fields, should I tamper with the `addclaims.py`. 
These logics are just unique for import. What do you suggest please?